### ISSUE TITLE
Custom sections missing

### ISSUE THREAD

**Issue by edgararuiz** [edgararuiz/ecodown#16]
https://tensorflow.rstudio.com/reference/keras/metric

**Comment by edgararuiz** [comment]
Being worked on here: https://github.com/edgararuiz/ecodown/tree/ref-qmd
